src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -58,7 +58,7 @@
         <h2>Business Dissolution Date</h2>
         <p class="mt-4 ">
           Enter the dissolution date of the business.
-          The dissolution date must be after the business start date and registration date.
+          The dissolution date must be on or after the registration date.
```

Comment:
Did Mihai approve this business logic change?

Does the Legal API need to validate this as well?

Could you please add some unit tests to verify the 3 date rule helpers with these edge cases?

Reply:
Riyaz opened the issue while Mihai had a look at it and did some changes in the ticket. So, I assumed that he already had a look at it and is OK with it. I'll double check and speak with him though Sev.

I don't think so as I don't know why, but the FE had an extra `+ 1` for the day here. So, I think it should be fine.

Sure thing! On it.

Reply:
Were you able to file a firm dissolution on the same date as the registration date? That would prove out the Legal API :)

Reply:
The `+1` was probably to make the day after (ie, after the business start date).

Reply:
Yup! I tested it.
![confirm date](https://github.com/bcgov/business-create-ui/assets/122301442/e08f38d5-3702-4784-86e6-0791a6ed2660)